vcpkg.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
   "name": "dosbox-staging",
   "description": "DOSBox Staging is a modern continuation of DOSBox with advanced features and current development practices.",
-  "version": "0.81.0-alpha",
+  "version": "0.82-rc",
```

Comment:
Minor: this belongs to the previous commit

Reply:
Fixed, I just stuck in a generic version since it's not really used by anything.